### PR TITLE
Dev gm checkapplications

### PR DESCRIPTION
**Pull request by GernotMaier**
Use this pull requests if the changes make sense in this images used mostly for testing and development:

- gammasim-tools (and simulation-model-description, verification-process if needed) are moved to a /external directory to keep the code out of the container 

- external/config.yml  - @orelgueta - why did we added this?

- add a script to to download the corsika/sim-telarray tar package from MIPK (password still needed)

Most of the changes have been added in October and I have a hard time to remember why. Happy to discard anything if necessary

### CODE REVIEW

**Comment by orelgueta** [comment]
> * external/config.yml  - @orelgueta - why did we added this?

This is the config file for gammasim-tools tailored to run within the container (all the paths point to workdir etc.).

File: docker-gammasim-tools-dev/external/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+mkdir -p verification-process
+cd verification-process
+git clone https://gitlab.cta-observatory.org/cta-consortium/aswg/simulations/simulation-model/verification/verification-process/mst-structure.git
+git clone https://gitlab.cta-observatory.org/cta-consortium/aswg/simulations/simulation-model/verification/verification-process/sst-structure.git
+git clone https://gitlab.cta-observatory.org/cta-consortium/aswg/simulations/simulation-model/verification/verification-process/mst-flashcam.git
+git clone https://gitlab.cta-observatory.org/cta-consortium/aswg/simulations/simulation-model/verification/verification-process/lst.git
+```
+3. instrument files (not yet on gitlab?)
+```
+from svn: dataFromInstrumentTeams
```
Comment (orelgueta) [inline docker-gammasim-tools-dev/external/README.md:27]: All instrument files are on gitlab, but it's a nightmare to download, so we can probably leave it like this...

**Review by orelgueta — APPROVED** [review]
Looks good

File: docker-gammasim-tools-dev/external/README.md (hunk 1 above)
Comment (GernotMaier) [reply docker-gammasim-tools-dev/external/README.md:27]: We need to solve this type of gitlab issues, but this is a general CTA / gitlab question.

**Pull request merged**